tests/Fixer/Phpdoc/PhpdocSeparationFixerTest.php

```diff
@@ -24,6 +24,11 @@
 {
     public function testFix()
     {
+        $this->doTest('<?php
+/** @param EngineInterface $templating 
```

Comment:
i would not treat it as proper docblock

still, tool must not crash on it, but i woulnd't say it's crucial to "fix" it